release/release_gold.sh

```diff
@@ -107,7 +107,10 @@ function main {
 
 	lc_time_run clean_portal_repository
 
-	lc_time_run prepare_next_release_branch
+	if [[ $(echo "${LIFERAY_RELEASE_PREPARE_NEXT_RELEASE_BRANCH}" | grep -i "yes") ]]
```

Comment:
@lucasmiranda0, do we really need double brackets? If not, can you please keep the single ones.

In addition, the function `update_release_info_date` is part of the prepare next functionality, therefore it should be also moved to be inside the scope of this `if`.

Thanks